beacon-chain/p2p/gossip_topic_mappings.go

```diff
@@ -15,6 +16,7 @@ var GossipTopicMappings = map[string]proto.Message{
 	"/eth2/voluntary_exit":     &pb.VoluntaryExit{},
 	"/eth2/proposer_slashing":  &pb.ProposerSlashing{},
 	"/eth2/attester_slashing":  &pb.AttesterSlashing{},
+	"/eth2/beacon_aggregate_and_proof": &pbp2p.AggregateAndProof{},
```

Comment:
This ideally should be a definition in `ethereumapis`, we'll need to update `ethereumapis` for the latest aggregate definitions 

Happy to do that first

Reply:
I think that's a good idea

Reply:
gofmt?

Reply:
will get to this tomorrow